src/schema/index.js

```diff
@@ -1,5 +1,13 @@
 import { schema } from 'normalizr';
 
-const movies = new schema.Entity('movies');
+const Movies = new schema.Entity('movies');
 
-export default movies;
+const Lists = new schema.Entity('lists');
+
+const Genres = new schema.Entity('genres');
+
+const Movie = new schema.Entity('movie', {
```

Comment:
You don't need separate schemes Movie and Movies, use 1 scheme. This will be helpful for denormalization because you don't need to remember which scheme you need to use.

Reply:
There is a small difference between these schemas.
shema Movie has a dependency - genres. shema Movies dont.

Reply:
That's fine, in this case normalizer just ignores this dependency and creates object without it.